docs/introduction/Examples.md

```diff
@@ -160,24 +123,6 @@ This example demonstrates rendering a deeply nested tree view and representing i
 
 This example includes tests.
 
-## Async
```

Comment:
Let's replace the "Async" entry here with the "Redux Essentials" example app example instead, including description and final source CodeSandbox.

Reply:
essentials too?